vardbg/output/video_writer/default_config.toml

```diff
@@ -45,3 +45,10 @@ heading_size = 24
 # Intro font, used to render the intro screen
 intro = "$ASSETS/fonts/Inter-Bold.ttf"
 intro_size = 48
+
+
+[theme]
+# Theme Name for Vardbg
```

Comment:
`for Vardbg` is a bit redundant and isn't really descriptive of what the option does. Something like `Color scheme to use` would convey the meaning better.

Reply:
Okay, I will update this one also.